src/compiler/crystal/types.cr

```diff
@@ -974,6 +987,16 @@ module Crystal
 
       nil
     end
+
+    def declare_class_var(name, type : Type, annotations = nil)
```

Comment:
I just noticed that `delcare_class_var` is only used in specs, so to avoid reviewing all the places where `ClassVarContainer` is included I would suggest moving this method to a private helper in the spec.

Reply:
Something like that?